powerauth-restful-security-spring-annotation/src/main/java/io/getlime/security/powerauth/rest/api/spring/provider/PowerAuthEncryptionProvider.java

```diff
@@ -57,8 +57,8 @@ public PowerAuthEciesDecryptorParameters getEciesDecryptorParameters(String acti
             eciesDecryptorRequest.setEphemeralPublicKey(ephemeralPublicKey);
             GetEciesDecryptorResponse eciesDecryptorResponse = powerAuthClient.getEciesDecryptor(eciesDecryptorRequest);
             return new PowerAuthEciesDecryptorParameters(eciesDecryptorResponse.getSecretKey(), eciesDecryptorResponse.getSharedInfo2());
-        } catch (Exception e) {
-            logger.warn("Get Ecies decryptor parameters call failed", e);
+        } catch (Exception ex) {
+            logger.warn("Get ECIES decryptor call failed, error: {}", ex.getMessage());
```

Comment:
Let's pass `ex` as a parameter to the new exception to keep the cause somewhere.